study.md

```diff
@@ -21,17 +21,19 @@ material. Instead, digest what you've read and repeat it in your own voice.
 Write the Ruby code that takes the integer `700` and returns the string `"007"`.
 
 ```ruby
-# your answer here
-```
+$ a = "007"
```

Comment:
`700.to_s.reverse`.  This will take the number 700, convert it to a string by invoking `to_s`, and then reverse it by invoking `reverse`.